editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -8576,7 +8576,8 @@ Node3DEditor::Node3DEditor() {
 	settings_fov->set_step(0.1);
 	settings_fov->set_value(EDITOR_GET("editors/3d/default_fov"));
 	settings_fov->set_select_all_on_focus(true);
-	settings_vbc->add_margin_child(TTR("Perspective FOV (deg.):"), settings_fov);
+	settings_fov->set_tooltip_text(TTR("The editor camera always uses the Keep Height aspect mode, so FOV is defined as a vertical value."));
```

Comment:
Forgot to insert in the review. The latter sentence is much more "of interest" to the user than the former.
```suggestion
	settings_fov->set_tooltip_text(TTR("FOV is defined as a vertical value. The editor camera always uses the Keep Height aspect mode."));
```
Or similar